backend/FwLite/MiniLcm.Tests/UpdateEntryTestsBase.cs

```diff
@@ -158,4 +158,67 @@ public async Task UpdateEntry_CanReorderSenses(string before, string after, stri
             actualOrderValues.Should().Be(expectedOrderValues);
         }
     }
+
+    [Theory]
+    [InlineData("a,b", "a,b,c,d", "1,2,3,4")] // append
+    [InlineData("a,2", "c,a,b", "0,1,2")] // single prepend
```

Comment:
this doesn't look right. First a,2 and c,a,b should result in 4 items as there's 4 distinct ids here. I'm guessing `a,2` is a typo and should be `a,b`. However due to this I would expect the test to fail as written, but it doesn't. So there's also a bug in the test.

Reply:
Good find. It was certainly unintentional and means this test wasn't actually just doing a "`single prepend`" like the comment claims, but the test itself is fine.